src/hexo/themes/documentation/helper/pagination.js

```diff
@@ -0,0 +1,147 @@
+module.exports = {
+    generate: (current, total, pattern, client, options) => {
+        const getFirst = function (c, t, p) {
+            return [{
+                active: false,
+                page: 1,
+                url: p.replace('pageId', 1)
+            }];
+        };
+
+        const getLast = function (c, t, p) {
```

Comment:
`c` isn't used.

@qzhou1607 Maybe refactor this to part to use better names for the parameters.